dpnp/dpnp_iface_trigonometric.py

```diff
@@ -2287,7 +2315,8 @@ def reduce_hypot(x, /, *, axis=None, dtype=None, keepdims=False, out=None):
     Input array, expected to have numeric data type.
 out : {None, dpnp.ndarray, usm_ndarray}, optional
     Output array to populate.
-    Array must have the correct shape and the expected data type.
+    Array must have a shape that the input broadcast to and
```

Comment:
```suggestion
    The array must have a shape to which the input can be broadcast and
```

However, it seems that `dpnp` (and `dpctl`) do not support broadcasting for unary functions.

```py
import numpy, dpnp
a = numpy.ones((2, 3))
out1 = numpy.empty((4, 2, 3))
numpy.tanh(a, out=out1).shape
# (4, 2, 3)

b = dpnp.ones((2, 3))
out2 = dpnp.empty((4, 2, 3))
dpnp.tanh(b, out=out2).shape
# ValueError: The shape of input and output arrays are inconsistent. Expected output shape is (2, 3), got (4, 2, 3)
```

Reply:
Nice point!
I have removed the changes for `out` parameter